app/models/form/sales/questions/buyer2_nationality.rb

```diff
@@ -0,0 +1,26 @@
+class Form::Sales::Questions::Buyer2Nationality < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "nationalbuy2"
+    @check_answer_label = "Buyer 2’s nationality"
+    @header = "What is buyer 2’s nationality?"
+    @type = "radio"
+    @answer_options = ANSWER_OPTIONS
+    @check_answers_card_number = 2
+    @question_number = 32
```

Comment:
this is here so that @natdeanlewissoftwire 's ticket 1864 (adding question numbers) can progress in parallel.
@natdeanlewissoftwire please confirm that this matches the desired format, although I feel I might have worked it out